fragdenstaat_de/fds_cms/templates/fds_cms/vega_chart.html

```diff
@@ -1,8 +1,10 @@
 {% load static %}
-{% load sekizai_tags %}
+{% load markup %}
 <div class="vega-chart">
     {% if instance.title %}<h5>{{ instance.title }}</h5>{% endif %}
-    {% if instance.description %}<p>{{ instance.description|linebreaksbr }}</p>{% endif %}
-    <div data-vegachart data-vegachartdata="{{ instance.vega_json }}"></div>
+    {% if instance.description %}{{ instance.description|markdown }}{% endif %}
```

Comment:
Does this break existing descriptions (e.g. ones which contain two asterisks from `*innen` or similar?)

Reply:
nope, we're in luck. only `:innen` 😉

Reply:
but should probably add a `help_text` indicating it's a markdown field